data_scrapers/san_mateo_county/assertions.py

```diff
@@ -0,0 +1,29 @@
+import re
+from bs4.element import Tag
+from typing import Dict, List
+
+class Assertions:
+    """
+    Class to encapsulate assertions.
+
+    These are mostly designed to ensure the webpage hasn't changed and was loaded correctly.
+    """
+    def iframes_match(self, iframes: List[Tag]) -> None:
+        if len(iframes) != 5:
+            raise FutureWarning('The number of dashboards on the start page has changed. It was 5.')
+
+        if sum('https://app.powerbigov.us' in iframe['src'] for iframe in iframes) != 4:
+            raise FutureWarning('iframes no longer recognized, check contents of page at START_URL.')
```

Comment:
We actually have a special error for cases like this called `FormatError` that we should be using instead of `FutureWarning`. Check the `sonoma` branch or the  news scrapers for an example

Reply:
Not applicable.